website/sidebars.js

```diff
@@ -253,7 +240,32 @@ module.exports = {
   },
   reference: {
     Configurations: [
-      "reference/dbt_project.yml",
+      {
+        type: "category",
+        label: "Project configurations",
```

Comment:
What do you think about pulling the `dbt_project.yml` file into its own link about the "Project configurations" section of the docs? That would look like:

```
- dbt_project.yml
- Project configurations
  - analysis-paths
  - clean-targets
  - ....
```

The benefit here is just that you could click on the `dbt_project.yml` link without expanding the section containing very-many project configs. I think we should also investigate reducing the padding on these sidebar links, as they take up a lot more visual space than they need to!

Reply:
This is what reduced padding could look like:

<img width="1059" alt="Screen Shot 2020-04-16 at 10 20 51 AM" src="https://user-images.githubusercontent.com/1541139/79467650-001a4900-7fcc-11ea-896b-a7f7f831e0b5.png">

Reply:
I buy that both of there are a step in the right direction!

Doing the first one now, can you commit the second part?